var/spack/repos/builtin/packages/blaspp/package.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+# 1) The CMake options exposed by `blaspp` allow for a value called `auto`. The
+#    value is not needed here as the choice of dependency in the spec
+#    determines the appropriate flags.
+#
+# 2) BLASFinder.cmake handles most options. For `auto`, it searches all blas
+#    libraries listed in `def_lib_list`.
+#
+# 3) ?? Custom blas library can be supplied via `BLAS_LIBRARIES`.
+#
+class Blaspp(CMakePackage):
+    """BLAS++: C++ API for the Basic Linear Algebra Subroutines (University of
+    Texas)."""
+
+    homepage = "https://bitbucket.org/icl/blaspp"
+    hg       = "https://bitbucket.org/icl/blaspp"
+    maintainers = ['teonnik', 'Sely85']
+
+    version('develop', hg=hg, revision="5191c9d")
+
+    variant('gfort',
+            default=False,
+            description=('Use GNU Fortran interface. '
+                         'Default is Intel interface. (MKL)'))
+    variant('ilp64',
+            default=False,
+            description=('Use 64bit integer interface. '
+                         'Default is 32bit. (MKL & ESSL)'))
+    variant('parallel',
+            default=False,
+            description=('Use OpenMP threaded backend. '
+                         'Default is sequential. (MKL & ESSL)'))
+
+    depends_on('blas')
+
+    def cmake_args(self):
+        spec = self.spec
+        args = ['-DBLASPP_BUILD_TESTS=OFF']
```

Comment:
Users can control this via `spack install --test=root` from the command line:
```suggestion
        args = ['-DBLASPP_BUILD_TESTS:BOOL={0}'.format(
            'ON' if self.run_tests else 'OFF')]
```
Do you know if this triggers further dependencies? For example, if test builds depend on [Catch2](https://github.com/catchorg/Catch2) v2.6.1+:
```py
    depends_on('catch2@2.6.1:', type='test')
```